src/transformers.jl

```diff
@@ -9,6 +9,15 @@ abstract type Transform end
 # Make Transforms callable types
 (t::Transform)(x; kwargs...) = apply(x, t; kwargs...)
 
+"""
+    is_transformable(x)
+
+
+Determine if `x` is transformable, which means that you can apply any `transform` to `x `.
+Currently Tables and AbstractArrays are transformable.
```

Comment:
```suggestion
Determine if `x` is both a valid input and output of any [`Transform`](@ref), which defines 
the [`transform`](@ref) interface.
Currently, all subtypes of `Table`s and `AbstractArray`s are transformable.
```

Reply:
The docstring for `transform` and `transform!` probably needs updating as well.

Reply:
> which defines the transform interface

I don't know what `which` refers to - this function, `Transform`, or being a valid input and output of any `Transform`. Could clarify that.